### PR TITLE
yaml-cpp updates

### PR DESCRIPTION
**Pull request by hjanetzek**
It works and all tests pass. @matteblair it seems to be using your CI system, could you have a look? IMO we could ignore this for now.

This change is needed to avoid superfluous copies of scene string.

### CODE REVIEW

**Comment by matteblair** [comment]
Removed the Appveyor CI webhooks. I remember now that I added them to test Windows fixes, but those were never merged.

**Review by tallytalwar — APPROVED** [review]
This look good to me.

**Comment by hjanetzek** [comment]
No need to merge this branch - I've tagged it with turbo-yaml-0.6.2.1

**Comment by matteblair** [comment]
@hjanetzek I think we should merge this into master anyway - it'll probably get confusing to keep track of more than one branch.

**Comment by tallytalwar** [comment]
Also whats up with the commits history here?

**Comment by hjanetzek** [comment]
@tallytalwar I've fixed bugs from the past where they were introduced and merged master back into turbo-yaml.. Not pretty I know - but I would like to have a bug-free commit series stacked on top of jbeder/master

**Comment by tallytalwar** [comment]
Gotcha.. In that case can you summarize the previous bugs and apt fixes please? Thanks
